examples/simple_example/main.tf

```diff
@@ -14,13 +14,47 @@
  * limitations under the License.
  */
 
-provider "google" {
-  version = "~> 2.0"
+provider "google-beta" {
+  version = "~> 3.0"
+  # Required if using User ADCs (Application Default Credentials) for Cloud Identity API.
+  user_project_override = true
+  billing_project       = var.project_id
+}
+
+resource "google_service_account" "owner" {
+  account_id = "example-owner-sa"
+  project    = var.project_id
+}
+
+resource "google_service_account" "manager" {
+  account_id = "example-manager-sa"
+  project    = var.project_id
+}
+
+resource "google_service_account" "member" {
+  account_id = "example-member-sa"
+  project    = var.project_id
+}
+
+module "child_group" {
+  source = "../.."
+
+  id           = "example-child-group@${var.domain}"
+  display_name = "example-child-group"
+  description  = "Example child group"
+  domain       = var.domain
+  owners       = ["${google_service_account.owner.account_id}@${var.project_id}.iam.gserviceaccount.com"]
```

Comment:
This could be a race condition since there is no dependency on creation of `google_service_account`. Could we use `google_service_account.foo.email` instead?

Reply:
Ah, I see. You cant depend on a computed attribute due to for_each within module.

Reply:
I think `google_service_account.owner.account_id` also imposes the dependency implicitly even the `account_id` field is not computed.